src/TurboMqtt/Protocol/ClientAcksActor.cs

```diff
@@ -24,6 +24,12 @@ namespace TurboMqtt.Protocol;
 /// </summary>
 internal sealed class ClientAcksActor : UntypedActor, IWithTimers
 {
+    public sealed class Reconnect
+    {
+        public static readonly Reconnect Instance = new();
+        private Reconnect() { }
+    }
+    
```

Comment:
Not sure if this is the correct place to put this message class

Reply:
Looks fine to me